src/util.cpp

```diff
@@ -121,21 +123,63 @@ void rocm_memory_lock_to_fine_grain(void* ptr, size_t size, void** gpu_ptr,
   }
 }
 
-struct rocshmem_env_config_t rocshmem_env_config;
 
-void rocshmem_env_config_init(void) {
+rocshmem_env_config::rocshmem_env_config() {
   char* env_value = NULL;
 
   env_value = getenv("ROCSHMEM_RO_DISABLE_IPC");
   if (NULL != env_value) {
-    rocshmem_env_config.ro_disable_ipc = atoi(env_value);
+    ro_disable_ipc = atoi(env_value);
   }
 
   env_value = getenv("ROCSHMEM_RO_PROGRESS_DELAY");
   if (nullptr != env_value) {
-    rocshmem_env_config.ro_progress_delay = atoi(env_value);
+    ro_progress_delay = atoi(env_value);
+  }
+
+  env_value = getenv("ROCSHMEM_BOOTSTRAP_TIMEOUT");
+  if (nullptr != env_value) {
+    bootstrap_timeout = atoi(env_value);
+  }
+
+  env_value = getenv("ROCSHMEM_HOSTID");
+  if (nullptr != env_value) {
+    hostid = std::string(env_value);
+  }
+
+  env_value = getenv("ROCSHMEM_SOCKET_FAMILY");
```

Comment:
optional: change all these to `ROCSHMEM_BOOTSTRAP_SOCKET_FAMILY`, `BOOTSTRAP_SOCKET_IFNAME`, `ROCSHMEM_BOOTSTRAP_HOSTID`

Reply:
since CI is not yet done, I will implement the suggested namechanges, they make sense